app/views/layouts/_navbar.html.erb

```diff
@@ -11,10 +11,10 @@
       <div class="collapse navbar-collapse" id="navbarCollapse">
          <ul class="navbar-nav me-auto mb-2 mb-md-0">
             <li class="nav-item">
-               <a class="nav-link" href="#">GPW</a>
+               <%= link_to 'GPW', root_path(query: 'SelectGPW'), class: "nav-link" %>
```

Comment:
I don't like using `query` to 2 things at the same time (search pattern & filtering out GPW/NC companies).

I would add another param instead and filter companies by 2 params:

```
/companies?query=alior&martket=gpw
```

Reply:
Ok, I can separate this, there will be 2 parameters, but I'm not sure if the way of passing `params` is what you had in mind?